app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -552,6 +554,19 @@ class MigrationsProvider(val context: Context) {
         }
     }
 
+    val MIGRATION_42_TO_43: Migration = object : Migration(42, 43) {
+        val fireproofLoginDetector = OldSettingsDataStore()
+
+        override fun migrate(database: SupportSQLiteDatabase) {
+            fireproofLoginDetector.updateFireproofSettingType()
+
+            database.execSQL(
+                "UPDATE $USER_STAGE_TABLE_NAME SET appStage = \"${AppStage.ESTABLISHED}\" " +
+                    "WHERE appStage = \"${AppStage.DAX_ONBOARDING}\""
+            )
```

Comment:
I think this was possibly copied by accident from `MIGRATION_41_TO_42` (Or is there a reason we need to skip the onboarding stage?)